src/Mono.Android/Mono.Android.targets

```diff
@@ -260,4 +268,37 @@
       <FileWrites Include="$(IntermediateOutputPath)CheckApiCompatibility.stamp" />
     </ItemGroup>
   </Target>
+
+  <Target Name="UpdateExternalDocumentation">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)Mono.Android.csproj"
+        Properties="TargetFramework=monoandroid10"
+        Targets="_UpdateExternalDocumentation"
+    />
+  </Target>
+  <Target Name="_UpdateExternalDocumentation">
+    <Exec
+        Command="git clean -xdf obj"
+        WorkingDirectory="$(MSBuildThisFileDirectory)"
+    />
+    <PropertyGroup>
+      <_Binlog>UpdateExternalDocumentation-$([System.DateTime]::Now.ToString ("yyyyMMddTHHmmss")).binlog</_Binlog>
+    </PropertyGroup>
+    <Exec
+        Command="msbuild /restore /p:IncludeAndroidJavadoc=True /bl:$(_Binlog)"
+        IgnoreStandardErrorWarningFormat="True"
+        WorkingDirectory="$(MSBuildThisFileDirectory)"
+    />
```

Comment:
Could this use `<MSBuild/>`? or does it actually need to run in a new process?

Reply:
I think the intent here was to make sure we always capture a unique binlog from the output. I've kept this for non windows platform, but updated this target to use <MSBuild/> on Windows so that it can be run there if desired.